index.html

```diff
@@ -0,0 +1,695 @@
+                                <img src="./assests/svgs/explore.svg" alt="">
+                                <span>Explore</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="notifications">
+                        <a href="/notifications.html">
+                            <div>
+                                <img src="./assests/svgs/notifications.svg" alt="">
+                                <span>Notifications</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="messages">
+                        <a href="/messages.html">
+                            <div>
+                                <img src="./assests/svgs/messages.svg" alt="">
+                                <span>Messages</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="bookmarks">
+                        <a href="/bookmarks.html">
+                            <div>
+                                <img src="./assests/svgs/bookmarks.svg" alt="">
+                                <span>Bookmarks</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="lists">
+                        <a href="/lists.html">
+                            <div>
+                                <img src="./assests/svgs/lists.svg" alt="">
+                                <span>Lists</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="profile">
+                        <a href="/profile.html">
+                            <div>
+                                <img src="./assests/svgs/profile.svg" alt="">
+                                <span>Profile</span>
+                            </div>
+                        </a>
+                    </div>
+                    <div class="more">
+                        <a href="/more.html">
+                            <div>
+                                <img src="./assests/svgs/more.svg" alt="">
+                                <span>More</span>
+                            </div>
+                        </a>
+                    </div>
+                </nav>
+                <div class="post" id="post">
+                    <a href="/post.html">
+                        <div>
+                            <h1>Post</h1>
+                        </div>
+                    </a>
+                </div>
+                <div class="userinfo">
+                    <a href="./profile.html">
+                        <div>
+                            <img src="./assests/images/SteamOxAvatar.png" alt="">
+                            <div>
+                                <h1>Freak</h1>
+                                <h2 class="at">@MaybeFreak</h2>
```

Comment:
what does this @ do and what is it selecting

Reply:
it's just part of the string it's not selecting anything